contrib/systemd/matrix-synapse.service

```diff
@@ -23,7 +26,7 @@ User=synapse
 Group=nogroup
 
 WorkingDirectory=/opt/synapse
```

Comment:
why is `workingdirectory` set to `/opt/synapse` if we're assuming that synapse has been installed in `/home/synapse/synapse` ?

Reply:
Good catch, I've submitted another PR addressing the issue.

Reply:
https://github.com/matrix-org/synapse/pull/6490